nx/lib/nx/random.ex

```diff
@@ -0,0 +1,374 @@
+defmodule Nx.Random do
+  @moduledoc """
+
+  ## PRNG
+  (Pseudo-Random Number Generator)

```

Comment:
```suggestion
  ## PRNG (Pseudo-Random Number Generator)

```

The intent is so that we don't introduce an acronym without it's meaning directly associated to it